main.py

```diff
@@ -1 +1,25 @@
 #Mad Libs Generator Project
+#Mad Libs Generator Project
+
+#Loop back to this point once code finishes
+loop = 1
+while (loop < 9):
```

Comment:
The loop is a great idea! With this current loop we will only get 8 iterations! 
try 
```python
while (loop <=10):
```